tests/tt_metal/tt_metal/unit_tests_common/basic/test_device_init.cpp

```diff
@@ -62,7 +62,7 @@ bool load_all_blank_kernels(tt_metal::Device* device) {
 
 INSTANTIATE_TEST_SUITE_P(DeviceInit, DeviceParamFixture, ::testing::Values(1, tt::tt_metal::GetNumAvailableDevices()));
 
-TEST_P(DeviceParamFixture, DeviceInitializeAndTeardown) {
+TEST_P(DeviceParamFixture, DISABLED_DeviceInitializeAndTeardown) { // see issue #6659
```

Comment:
Noticed that this only fails on N300 according to issue
 If so, we shouldn't use `DISABLED_` and skip if device is N300

Reply:
when i was iterating the test will try to run and hang if i try using the skip, so i just disabled it completely